.ci/Jenkinsfile

```diff
@@ -94,8 +94,9 @@ pipeline {
             script {
               def ruby = readYaml(file: '.ci/.jenkins_ruby.yml')
               def testTasks = [:]
+              def i = 0
               ruby['RUBY_VERSION'].each{ rubyVersion ->
```

Comment:
You can use `eachWithIndex` http://docs.groovy-lang.org/latest/html/groovy-jdk/java/lang/Object.html#eachWithIndex(groovy.lang.Closure)

Reply:
I'd recommend to not use it, this is not groovy, it is groovy-ish, you have to try to use as lees as possible grovy extensions and keep the code on basics groovy features to avoid problems with the script security plugin and the DSL implementation, also it probably isn't implemented on the DSL plugin.